doc/release/1.17.0-notes.rst

```diff
@@ -191,6 +214,17 @@ This change is backwards compatible, but now allows code like::
 New Features
 ============
 
+New extensible random module with selectable random number generators
+---------------------------------------------------------------------
+  A new extensible random module along with four selectable random number
+  generators and improved seeding designed for use in parallel processes has
+  been added. The currently available bit generators are MT19937, PCG64,
+  Philox, and SFC64. PCG64 is the new default while MT19937 is retained for
+  backwards compatibility. Note that the legacy random module is unchanged and
+  is now frozen, your current results will not change. Extensive documentation
```

Comment:
It would be great to dd a link here referencing the new module in our sphinx docs.

Reply:
I'd like to do that, are the docs up?

Reply:
It's on devdocs, not scipy.org, but there is www.numpy.org/devdocs/reference/random/new-or-different.html

Reply:
Can we just use internal sphinx links here?